### DISCUSSION TITLE
Requesting a fixture in a pytest.Item item

### DISCUSSION THREAD

**Discussion by GergelyKalmar** [pytest-dev/pytest#10915] [Q&A]
I'm trying to request a specific fixture in a pytest.Item object that is part of a plugin (something similar to this: https://github.com/logikal-io/pytest-logikal/blob/main/pytest_logikal/licenses.py#L56), however, I couldn't find a way to actually do that. I don't need the return value of the fixture, I just need the side effect to take place.

I've found out that I can add marks in the `__init__` function via `self.add_marker('xxx')`, however, I couldn't find a way to make the item request a fixture before `runtest` runs (and call the teardown when `runtest` is done). Is that possible somehow? I tried to call the fixture directly, but that gave me an error:

```
Fixture "xxx" called directly. Fixtures are not meant to be called directly, but are created automatically when test functions request them as parameters. See https://docs.pytest.org/en/stable/explanation/fixtures.html for more information about fixtures, and
https://docs.pytest.org/en/stable/deprecations.html#calling-fixtures-directly about how to update your code.
```

**Comment by RonnyPfannschmidt** [comment]
Currently fixtures are not extended to items in the general sense
